src/main/java/org/apache/maven/shared/verifier/Verifier.java

```diff
@@ -1263,12 +1301,25 @@ public void executeGoals( List<String> goals, Map<String, String> envVars )
             args.add( "-Dmaven.repo.local=" + localRepo );
         }
 
-        args.addAll( allGoals );
+        if ( autoclean )
+        {
+            /*
+             * NOTE: Neither test lifecycle binding nor prefix resolution here but call the goal directly.
+             */
+            args.add( "org.apache.maven.plugins:maven-clean-plugin:clean" );
```

Comment:
Would it make sense to move this value to a static final for documentation puposes?

Reply:
ok, try with constant ....